tests/system/action/mediafile/test_upload.py

```diff
@@ -399,7 +399,7 @@ def test_upload_orga_owner_permissions(self) -> None:
             OrganizationManagementLevel.CAN_MANAGE_ORGANIZATION,
         )
 
-    def test_create_added_mimetype_ttf(self) -> None:
+    def test_incoherent_mimetype_error(self) -> None:
```

Comment:
if you want you could delete the test. It's the same case like `test_create_cannot_guess_mimetype`

Reply:
Done